### ISSUE TITLE
Is there a way to detect if the device can actualy vibrate?

### ISSUE THREAD

**Issue by nhoizey** [Modernizr/Modernizr#1217]
`feature-detects/vibrate.js` detects if the vibrate method is available in the browser, not if the device can actualy vibrate.

https://developer.mozilla.org/en-US/docs/Web/API/Navigator.vibrate states that _"if the device doesn't support vibration, this method has no effect"_, but doesn't say anything about actual vibration support.

**Comment by dephraiim** [comment]
This worked for me. It checked if the Vibration API was supported in the current browser.

```js
function canDeviceVibrate() {
  return "vibrate" in navigator;
} 
```

**Comment by roblarsen** [comment]
@dephraiim that's the whole point of this issue (which I just realized I took part in more than 7 years ago)  Testing for the `vibrate` method in `navigator` doesn't actually tell you anything about the device. It tests if the API is present.

Here are my windows 8 laptop tests from 2014. The API was present in Chrome and Firefox, but the device didn't vibrate.  Still the case today. 
👇 

> > Pretty sure my MBA can’t vibrate and 'vibrate' in navigator is true in Chrome.
> 
> Chrome Win 8 : true 
> Firefox Win8 : true 
> IE10 Win8: false
